101-azurefirewall-create-with-firewallpolicy-apprule-netrule-ipgroups/README.md

```diff
@@ -0,0 +1,8 @@
+This template deploys an Azure Firewall with Firewall Policy (including multiple application and network rules) referencing IP Groups in application and network rules.
```

Comment:
See contribution guide for a proper README.md example

Reply:
Changes has been made to README file as per guidance.